scene/2d/camera_2d.h

```diff
@@ -123,6 +129,22 @@ class Camera2D : public Node2D {
 	void _validate_property(PropertyInfo &p_property) const;
 
 public:
+#ifdef TOOLS_ENABLED
+	virtual Dictionary _edit_get_state() const override;
+	virtual void _edit_set_state(const Dictionary &p_state) override;
+
+	virtual void _edit_set_position(const Point2 &p_position) override;
+	virtual Point2 _edit_get_position() const override;
+
+	virtual void _edit_set_rect(const Rect2 &p_rect) override;
+	virtual Size2 _edit_get_minimum_size() const override { return Size2(); }
+#endif // TOOLS_ENABLED
+
+#ifdef DEBUG_ENABLED
+	virtual Rect2 _edit_get_rect() const override;
+	virtual bool _edit_use_rect() const override;
+#endif // DEBUG_ENABLED
```

Comment:
I see we have code like this in multiple classes.
I wonder why it's not inside TOOLS_ENABLED.